x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -271,7 +277,12 @@ export class EndpointDocGenerator {
    * Creates new random policy id for the host to simulate new policy application
    */
   public updatePolicyId() {
-    this.commonInfo.endpoint.policy.id = this.randomChoice(POLICIES).id;
+    this.commonInfo.endpoint.policy.applied.id = this.randomChoice(POLICIES).id;
```

Comment:
nit: Maybe change the name of this function since it's updating more than just the policy id right?